pypeit/utils.py

```diff
@@ -480,6 +480,16 @@ def bspline_profile(xdata, ydata, invvar, profile_basis, inmask = None, upper=5,
     return sset, outmask, yfit, reduced_chi, exit_status
 
 
+def cap_ivar(flux, ivar, sn_cap, mask=None):
```

Comment:
`clip_ivar` to give similar connotation to `np.clip`?

Reply:
Good idea, although it does not exactly clip, it is adding error.

Reply:
Changed.